### PR TITLE
Send extra properties in object creation or update

### PR DESCRIPTION
**Pull request by SferaDev**

### CODE REVIEW

File: packages/client/src/util/fetch.ts [hunk 1]
```diff
@@ -11,3 +24,70 @@ export function getFetchImplementation(userFetch?: FetchImpl) {
...
+
+    return this.#fetch;
+  }
+
+  request(url: string, options?: RequestInit): Promise<Response> {
+    const fetch = this.getFetch();
+
+    return this.#enqueue(async () => {
+      const response = await fetch(url, options);
+      // TODO: Check headers
```
Comment (SferaDev) [inline packages/client/src/util/fetch.ts:61]: Will follow-up in a new PR

File: packages/client/src/schema/repository.ts [hunk 2]
```diff
@@ -1655,15 +1667,29 @@ export class RestRepository<Record extends XataRecord>
   }
 }
 
-const transformObjectLinks = (object: any) => {
+function cleanSaveRecord<T>(object: any, tableColumns: Schemas.Column[]): T {
```
Comment (gimenete) [inline packages/client/src/schema/repository.ts:1670]: I'm not sure about this. The fact that we ignore fields that are not in the database could be a source of confusion.

I think we can maybe expose this function with a different name and allow users to use it explicitely, but not do it internally ourselves.

WDYT?

Comment (SferaDev) [reply packages/client/src/schema/repository.ts:1670]: I would like to avoid adding even more methods, but an option `ignoreExtraProps` might be helpful

Comment (gimenete) [reply packages/client/src/schema/repository.ts:1670]: `ignoreExtraProps` works for me.

**Pull request closed without merging**